packages/relay-runtime/util/recycleNodesInto.js

```diff
@@ -30,29 +30,31 @@ function recycleNodesInto<T>(prevData: T, nextData: T): T {
   const prevArray = Array.isArray(prevData) ? prevData : null;
   const nextArray = Array.isArray(nextData) ? nextData : null;
   if (prevArray && nextArray) {
+    const isFrozen = Object.isFrozen(nextArray);
```

Comment:
Since we only freeze in DEV, perhaps we should only perform this isFrozen check in dev?
cc @leebyron

Reply:
Ya, I thought about that too but figured isFrozen should be pretty darn quick, and there’s no reason to try to modify a frozen object ever. Happy to add that though.

Reply:
Sounds good, I can go ahead and add that tiny change before landing this PR if that sounds good to you!

Reply:
👍 Sure!

Reply:
Hey, not sure if you already made the change over at FB, but I added the dev check here. Let me know if there's anything more I can do to help get this merged...